tests/cli/test_main_rename.py

```diff
@@ -189,18 +182,15 @@ def test_rename_with_force_with_errors(env_one, env_two):
     # Do a force rename
     with mock.patch("conda.cli.main_rename.install.clone") as clone_mock:
         clone_mock.side_effect = [CondaError(error_message)]
-        out, err, exit_code = run(
+        _, err, exit_code = run(
             f"conda rename -n {TEST_ENV_NAME_1} {TEST_ENV_NAME_2} --force", disallow_stderr=False
         )
         assert error_message in err
-
-    (out, err, exit_code), data = list_envs()
-    result = data.get("envs", [])
+        assert exit_code == 1
 
     # Make sure both environments still exist
     assert locate_prefix_by_name(TEST_ENV_NAME_2)
     assert locate_prefix_by_name(TEST_ENV_NAME_1)
-    assert err == ""
```

Comment:
instead of removing still keep it to check for if there is an error in the environment. assert error is None

Reply:
`def run_inprocess_conda_command(command, disallow_stderr: bool = True): ... return c.stdout, c.stderr, exit_code`

Since we are outputting the new message to stderr, `err` will contain that message instead of the empty string.